src/Microsoft.DotNet.XHarness.Tests.Runners/iOSApplicatonEntryPoint.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.DotNet.XHarness.Tests.Runners.xUnit;
+
+namespace Microsoft.DotNet.XHarness.Tests.Runners.Core
+{
+    public abstract class iOSApplicatonEntryPoint : ApplicationEntryPoint
+    {
+        public override async Task RunAsync()
+        {
+            var options = ApplicationOptions.Current;
+            TcpTextWriter writer = null;
+            if (!string.IsNullOrEmpty(options.HostName))
+            {
+                try
+                {
+                    writer = new TcpTextWriter(options.HostName, options.HostPort);
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine("Network error: Cannot connect to {0}:{1}: {2}. Continuing on console.", options.HostName, options.HostPort, ex);
+                    writer = null; // will default to the console
+                }
+            }
+
+            // we generate the logs in two different ways depending if the generate xml flag was
+            // provided. If it was, we will write the xml file to the tcp writer if present, else
+            // we will write the normal console output using the LogWriter
+            var logger = (writer == null || options.EnableXml) ? new LogWriter(Device) : new LogWriter(Device, writer);
+            logger.MinimumLogLevel = MinimumLogLevel.Info;
+            var testAssemblies = GetTestAssemblies();
+            TestRunner runner;
+            switch (TestRunner)
+            {
+                case TestRunnerType.NUnit:
+                    throw new NotImplementedException("The NUnit test runner has not yet been implemened.");
```

Comment:
spelling (implemented) also that sentence is kind of implied by the exception type.